src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -264,6 +322,56 @@ private void validateSsl() {
     }
   }
 
+  private void validateVersion(RestHighLevelClient client) {
+    if (!config.isDataStream()) {
+      return;
+    }
+    MainResponse response;
+    try {
+      response = client.info(RequestOptions.DEFAULT);
```

Comment:
can you please find out what permissions are needed for this API and add them to our documentation

Reply:
Resolved in 1:1. The connector would need the `monitor` privilege for the ping and info APIs. The document has been updated by the docs team to reflect this ([PR link](https://github.com/confluentinc/docs-kafka-connect-elasticsearch/pull/47))